spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -111,12 +104,26 @@ If an annotation is not present, the configured properties are ignored.
 For instance, the property `com.acme.ClassA/methodB/Retry/maxRetries` will be ignored if `@Retry` annotation is not specified on the `methodB` of `com.acme.ClassA`.
 Similarly, the property `com.acme.ClassA/Retry/maxRetries` will be ignored if `@Retry` annotation is not specified on the class `com.acme.ClassA` as a class-level annotation.
 
-=== Disabled Fault Tolerance policies with configuration
+=== Disable a group of Fault Tolerance annotations on the global level
+In some service mash platform, e.g. Istio, has its own Fault Tolerance policy.
+The operation team might want to use the platform Fault Tolerance.
+In order to fulfil the requirement, MicroProfile Fault Tolerance provides a capability to have its resilient functionalities except `fallback` disabled.
+The reason `fallback` is special as the `fallback` business logic can only be defined by microservices not by any other platforms.
```

Comment:
"is special **is that the** `fallback`" ... "by microservices **and not by** any other" - grammar fixes